### PR TITLE
style: style/performance improvement for condition query

### PR DESCRIPTION
**Pull request by Anna-MariiaPetryk**
Closes: #763

### CODE REVIEW

**Comment by Anna-MariiaPetryk** [comment]
closed#763

**Comment by Anna-MariiaPetryk** [comment]
closes #763

**Review by dshovchko — APPROVED** [review]

File: src/modules/esl-media/core/esl-media.ts [hunk 1]
```diff
@@ -376,10 +371,10 @@ export class ESLMedia extends ESLBaseElement {
 
   // TODO: simplify - ESLMediaQuery is lazy for primitives, it make sense to init it unconditionally
```
Comment (ala-n) [inline src/modules/esl-media/core/esl-media.ts:373]:
```suggestion
```

**Review by ala-n — APPROVED** [review]

**Review by julia-murashko — APPROVED** [review]

File: src/modules/esl-media/core/esl-media.ts [hunk 2]
```diff
@@ -75,6 +75,8 @@ export class ESLMedia extends ESLBaseElement {
   @attr() public loadClsAccepted: string;
   /** Class / classes to add when media is declined */
   @attr() public loadClsDeclined: string;
+  /** load-condition . Default: `all` */
```
Comment (abarmina) [inline src/modules/esl-media/core/esl-media.ts:78]: extra cosmetic (smt like that):
```suggestion
  /** Media loading conditions. Default: `all` */
```

**Review by abarmina — APPROVED** [review]

File: src/modules/esl-media/core/esl-media.ts (hunk 2 above)
Comment (ala-n) [reply src/modules/esl-media/core/esl-media.ts:78]: Fixed in #778

**Pull request merged**